cmd/operator-sdk/generate/internal/genutil.go

```diff
@@ -67,7 +67,7 @@ func PluginKeyToOperatorType(pluginKey string) projutil.OperatorType {
 }
 
 // WriteCRDs writes each CustomResourceDefinition in crds to w.
-func WriteCRDs(w io.Writer, crds ...v1beta1.CustomResourceDefinition) error {
+func WriteCRDs(w io.Writer, crds ...apiextv1.CustomResourceDefinition) error {
```

Comment:
I am a little worried that writing a v1 CRD when the one being copied could be v1beta1 will break things. Need to make sure conversion always generates a valid manifest.

Reply:
By handling both v1 and v1beta1 CRDs explicitly we avoid this issue, although with more verbose code.